namespacecost.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+
+	"github.com/gorilla/mux"
+)
+
+func (s *server) GetCost(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	namespace := vars["namespace"]
+
+	log.Printf("loading namespace costs for %q", namespace)
+
+	params := url.Values{}
+	params.Add("namespace", namespace)
+
+	resp, err := http.Get("http://localhost:9090/model/aggregatedCostModel?window=1d&aggregation=namespace&" + params.Encode())
+
+	if err != nil {
+		s.error(w, r, err)
+		return
+	}
+	defer resp.Body.Close()
+
+	// Read the incoming cost information
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		s.error(w, r, err)
+		return
+	}
+
+	responseString := string(body)
+	var response interface{}
+	json.Unmarshal([]byte(responseString), &response)
+
+	s.respond(w, r, response)
```

Comment:
Don't use `respond` here as it doesn't do what you need. You're really just passing along the response from the Kubecost API, so just write out the status, content type, and body directly from the response.